README.md

```diff
@@ -1,17 +1,32 @@
 # Mapbox Annotation Extension for iOS
 
-The Mapbox Annotation Extension is lightweight library you can use with the Mapbox Maps SDK to quickly add basic shapes to a map.
+The Mapbox Annotation Extension is a lightweight library you can use with the Mapbox Maps SDK for iOS to quickly add basic shapes to a map.
 
-⚠️ This product is currently in active alpha development, and is not intended for usage in production. We welcome feedback
-from developers in issues. ⚠️
+This extension leverages the power of the Mapbox runtime styling api with an object oriented approach to simplify the creation and styling of annotations.
```

Comment:
We're trying to drop the usage of the word "Mapbox runtime styling api", my suggested replacement would be just "runtime styling".